contracts/FileStorage.sol

```diff
@@ -279,6 +293,12 @@ contract FileStorage is AccessControlEnumerableUpgradeable {
         require(success, "EVM error in getFileSize");
     }
 
+    function isImmutable(string calldata storagePath) external view returns (bool) {
+        (address owner, string memory contentPath) = Utils.parseStoragePath(storagePath);
+        (,DetailedInfo memory content) = getContentInfo(owner, contentPath);
```

Comment:
It's cheaper here not to load whole DelatiledInfo into memory but use
`DetailedInfo storage content` instead.